tox.ini

```diff
@@ -172,6 +172,11 @@ commands = python "{toxinidir}/tools/clean.py" "{toxinidir}/falcon"
 [with-cython]
 deps = -r{toxinidir}/requirements/tests
        Cython
+       # NOTE(vytas): By using --no-build-isolation, we need to manage build
```

Comment:
we need to drop that for v4. is there an issue for it?

Reply:
Now, there is: https://github.com/falconry/falcon/issues/2184.
I tentatively scheduled it for 4.0, but we might need to push down the road in favour of a timely release.